scripts/generic-ci-tests.sh

```diff
@@ -80,8 +80,8 @@ case "$TEST_SUITE" in
         paver run_pylint -l $PYLINT_THRESHOLD > pylint.log || { cat pylint.log; EXIT=1; }
 
         mkdir -p reports
-        echo "Finding jshint violations and storing report..."
-        paver run_jshint -l $JSHINT_THRESHOLD > jshint.log || { cat jshint.log; EXIT=1; }
+        echo "Finding ESLint violations and storing report..."
+        paver run_eslint -l $ESLINT_THRESHOLD > eslint.log || { cat eslint.log; EXIT=1; }
```

Comment:
Created [TE-1549](https://openedx.atlassian.net/browse/TE-1549) and [TE-1550](https://openedx.atlassian.net/browse/TE-1550), thanks for the heads up.